inc/admin/options.php

```diff
@@ -75,20 +75,23 @@ function rocket_after_save_options( $oldvalue, $value ) {
 		);
 	}
 
-	// Purge all minify cache files.
 	// phpcs:ignore WordPress.Security.NonceVerification.Missing
-	if ( ! empty( $_POST ) && ( $oldvalue['minify_css'] !== $value['minify_css'] || $oldvalue['exclude_css'] !== $value['exclude_css'] ) || ( isset( $oldvalue['cdn'] ) && ! isset( $value['cdn'] ) || ! isset( $oldvalue['cdn'] ) && isset( $value['cdn'] ) ) ) {
-		rocket_clean_minify( 'css' );
-	}
-
-	// phpcs:ignore WordPress.Security.NonceVerification.Missing
-	if ( ! empty( $_POST ) && ( $oldvalue['minify_js'] !== $value['minify_js'] || $oldvalue['exclude_js'] !== $value['exclude_js'] ) || ( isset( $oldvalue['cdn'] ) && ! isset( $value['cdn'] ) || ! isset( $oldvalue['cdn'] ) && isset( $value['cdn'] ) ) ) {
+	if ( ! empty( $_POST )
```

Comment:
`! empty( $_POST )` is used here and the next 2 conditional expressions. That's redundant and little bit slower. Instead, use it once and pull all 3 conditionals under it.

Reply:
I'm not sure why this check for `$_POST` was there in the first place (it's part of the very old code we have).

It is preventing the code inside the conditional from running if the update was not triggered from the settings page, but why would we want that? It should also be running if we call `update_option()` directly.

Does it make sense to you?

Reply:
Yes. I confirm that `$_POST` is from old code in there and can be safely removed. I kept it because it was out of this task scope. However, this condition should be moved in the Admin Subscriber from Engine/Optimization/Minify/JS when it will be revisited. 

If we decide to remove the POST i recommend to move it now in it's AdminSubscriber and write Unit & Integration tests. It should not take too much time.